src/style/app.module.css

```diff
@@ -676,7 +730,8 @@
 }
 
 .rowBackground {
-  background-color: var(--row-background);
+  background-color: var(--white-color);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`